pkg/beaconfqdn/dissector.go

```diff
@@ -46,14 +47,96 @@ func (d *dissector) close() {
 }
 
 //start kicks off a new analysis thread
+/*
+db.getCollection('uconn').aggregate([
+    {"$match": {
+        "$or": [{"dst":"172.217.4.226"}]
+    }},
+    {"$project": {
+        "src":              1,
+        "src_network_uuid": 1,
+        "src_network_name": 1,
+        "ts": {
+            "$reduce": {
+                "input":            "$dat.ts",
+                    "initialValue": [],
+                    "in":           {"$concatArrays": ["$$value", "$$this"]},
+            },
+        },
+        "bytes": {
+            "$reduce": {
+                "input":        "$dat.bytes",
+                "initialValue": [],
+                "in":           {"$concatArrays": ["$$value", "$$this"]},
+            },
+        },
+        "count":  {"$sum": "$dat.count"},
+        "tbytes": {"$sum": "$dat.tbytes"},
+    }},
+    {"$group": {
+        "_id":              {"src": "$src", "uuid": "$src_network_uuid", "network": "$src_network_name"},
```

Comment:
Looks like this comment needs to be changed to match the recent changes.

Reply:
I believe this PR is ready to pull in otherwise.

Reply:
Think i've got it fixed, thanks!